src/test/java/com/aws/greengrass/secretmanager/SecretManagerIPCAgentTest.java

```diff
@@ -1,82 +0,0 @@
-/*
```

Comment:
we don't need a test anymore?

Reply:
Removed as there's only one test in this file and that is already covered in integ tests with less mocking.